builder/common_builder/oracle_databases.go

```diff
@@ -356,7 +362,6 @@ func (b *CommonBuilder) getMountedDatabase(entry agentmodel.OratabEntry, host mo
 	database.PSUs = []model.OracleDatabasePSU{}
 	database.Backups = []model.OracleDatabaseBackup{}
 	database.PDBs = []model.OracleDatabasePluggableDatabase{}
-	database.Services = []model.OracleDatabaseService{}
```

Comment:
Would you mind to check if this is a needed field by our json schema?
https://github.com/ercole-io/ercole/blob/master/schema/oracle.json#L521

Reply:
This field is a needed field by our json schema, so it is mandatory and we need to restore it.